fuse_optimizers/src/optimizer.cpp

```diff
@@ -99,7 +99,7 @@ Optimizer::Optimizer(
       private_node_handle_.createTimer(ros::Duration(diagnostic_updater_timer_period_),
                                        boost::bind(&diagnostic_updater::Updater::update, &diagnostic_updater_));
 
-  diagnostic_updater_.add("Optimizer", this, &Optimizer::setDiagnostics);
+  diagnostic_updater_.add(private_node_handle_.getNamespace(), this, &Optimizer::setDiagnostics);
```

Comment:
This changes the name, and simplifies the code remove from the children `setDiagnostics()` method; the one the *diff* doesn't show well.